hybrasyl/CreatureStatus.cs

```diff
@@ -71,18 +73,22 @@ public bool Check(ConditionInfo condition)
             return false;
         }
     }
-
+    
     public interface ICreatureStatus
     {
 
         string Name { get; }
+        string CastableName { get; }
```

Comment:
A status should be independent of the castable. The castable should reference the status. Not sure why this is here.

Reply:
Included due to current accessibility in formulas. Also, the castable itself is needed for the `Intensity` value, though that could also be serialized to remove that specific dependency.

Reply:
Fair enough, though the point of statuses being separate was to remove the dependency on castables.

Reply:
Sure, this can be `String.Empty` though (note that the CreatureStatus allows this). Really, it is only needed if used in the formula. It is included here to handle the more generic case of a castable having a defined intensity on a reused status.